### PR TITLE
suberisedapps

### PR DESCRIPTION
**Pull request by gautam1512**

### CODE REVIEW

**Review by DrAmoled — CHANGES_REQUESTED** [review]
Please remove the archive path, so it will be 11/gautam.config instead of archive/11/gautam.config

**Comment by gautam1512** [comment]
I have done that now, is it okay?

**Comment by nikhilmenghani** [comment]
> I have done that now, is it okay?

You renamed sample.config to Gautam.config. Make sure you only change files in 11 or 10 folder. Better fork the repository and start again and send pull request.

**Comment by gautam1512** [comment]
Sir i have tried putting the ultimatum.config file in config/11 folder. I hope, i have done this right this time.

**Comment by nikgapps** [comment]
@gautam1512 Your pull request has 3 changes, there should only be 1 change per android version. Please fork the repository, add your config to 10 or 11 folder and send the pull request.

**Comment by gautam1512** [comment]
Okay sir I try again.

**Comment by nikhilmenghani** [comment]
@gautam1512 there should be no changes in archive folder. Kindly resend the pull request after making the changes. Only changes from 10 or 11 folder will be accepted

File: 11/subapps.config [hunk 1]
```diff
@@ -71,7 +74,7 @@ GoogleClock=1
 
 # Set SetupWizard=0 if you want to skip installing all packages belonging to SetupWizard Package
 SetupWizard=1
->>SetupWizard=1
+>>SetupWizard=0
```
Comment (DrAmoled) [inline 11/subapps.config:77]: SetupWizard need to have value of 1 even if you are using Pixel Setup Wizard

File: 11/ultimatum.config [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+VancedManager=0
+GoogleMessages=1
+GoogleDialer=1
+GoogleContacts=1
+CarrierServices=1
+GoogleClock=1
+
+# Set SetupWizard=0 if you want to skip installing all packages belonging to SetupWizard Package
+SetupWizard=1
+>>SetupWizard=0
```
Comment (DrAmoled) [inline 11/ultimatum.config:77]: Same as previous comment

File: 11/ultimatum.config [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+# Set PixelLauncher=0 if you want to skip installing all packages belonging to PixelLauncher Package
+PixelLauncher=1
+>>PixelLauncher=1
+>>DevicePersonalizationServices=1
+>>QuickAccessWallet=1
+
+
+# Set GoogleFiles=0 if you want to skip installing all packages belonging to GoogleFiles Package
+GoogleFiles=1
```
Comment (DrAmoled) [inline 11/ultimatum.config:108]: Why this appset is 1 when you put 0 in both sub appset

**Review by DrAmoled — CHANGES_REQUESTED** [review]
Adjustment needed according to the comments I put in your config

**Comment by nikhilmenghani** [comment]
@gautam1512 can you make the changes today?

I'll close the pull request if there is no response. Send the pull request again if you need.

**Pull request closed without merging**